tests/test_other.py

```diff
@@ -2216,7 +2216,12 @@ def test_debuginfo(self):
           (['-O2', '-g'], True),
         ]:
         print(args, expect_debug)
+        self.clear()
         err = run_process([PYTHON, EMCC, path_from_root('tests', 'hello_world.cpp')] + args, stdout=PIPE, stderr=PIPE).stderr
+        if '-g4' in args:
+          assert os.path.exists('a.out.wasm.map'), 'a source map should be emitted'
```

Comment:
I think we have self.assertExists() and self.assertNotExists() these days don't we?